src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/SpidersDenServerWidget.java

```diff
@@ -0,0 +1,86 @@
+package de.hysky.skyblocker.skyblock.tabhud.widget;
+
+import de.hysky.skyblocker.skyblock.tabhud.util.Ico;
+import de.hysky.skyblocker.skyblock.tabhud.util.PlayerListMgr;
+import net.minecraft.text.MutableText;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+import java.util.Arrays;
+
+/**
+ * This widget shows info about the Spider's Den server
+ */
+public class SpidersDenServerWidget extends Widget {
+
+    private static final MutableText TITLE =
+            Text.literal("Server Info").formatted(Formatting.DARK_AQUA, Formatting.BOLD);
+
+    /**
+     * Broodmother Mini-Boss tab states
+     */
+    private enum BroodmotherState {
```

Comment:
This is missing the "Imminent" state
<img width="320" alt="image" src="https://github.com/SkyblockerMod/Skyblocker/assets/51387595/29d94170-c1fd-4276-8545-64422f0be291">

Reply:
It's also missing the "Slain" state
<img width="269" alt="image" src="https://github.com/SkyblockerMod/Skyblocker/assets/51387595/d4a93e5d-ebe0-4053-9599-03fef5e16824">